neurokit2/signal/signal_detrend.py

```diff
@@ -43,8 +43,9 @@ def signal_detrend(
         Only used if ``method`` is "loess". The parameter which controls the degree of smoothing.
     window : float
         Only used if ``method`` is "locreg". The detrending ``window`` should correspond to the
-        desired low frequency band to remove (for instance, ``1.5`` will remove frequencies below
-        1.5Hz).
+        sample rate divided by the desired low frequency band to remove (window=sample_rate / frequency) .
+        For instance, to remove frequencies below 0.67Hz for a signal sampled at 1000Hz, a window
+        of the size round(1000 / 0.67) = ``1.5*1000`` should be used.
```

Comment:
With the current implementation in which `window` is multiplied by `sampling_rate`, I think the correct input for this example would be `(1000/0.67)/1000` rather than `round(1000 / 0.67)`. 

@DominiqueMakowski we could 
1. change the doc example to reflect the current implementation, 
2. change the implementation so that `window` isn't multiplied by `sampling_rate`, or 
3. change the implementation so that `lowcut` is an argument instead of `window` and compute `window` from `sampling_rate / lowcut` (assuming that the [pyprep implementation](https://github.com/sappelhoff/pyprep/blob/master/pyprep/removeTrend.py#L92) is actually incorrect, which I don't feel qualified to judge honestly)